ember-cli-build.js

```diff
@@ -75,6 +75,18 @@ module.exports = function () {
     },
     'ember-power-select': {
       theme: false
+    },
+    postcssOptions: {
+      compile: {
+        enabled: false
+      },
+      filter: {
+        enabled: true,
+        exclude: ['vendor/**/*'],
+        plugins: [
+          require('tailwindcss')('./app/styles/app/tailwind/config.js')
+        ]
+      }
```

Comment:
This is something different compared to [documentation](https://github.com/chrism/emberjs-tailwind-purgecss#including-in-the-build-pipeline), is it on purpose? I didn't dive deep into it, just noticed a difference :)

Reply:
Yeah, I tried that way first, and first ran into an issue where postcss was expecting `/app/styles/app.css` rather than `/app/styles/app.scss`. So I tried to follow this [guide](https://github.com/jeffjewiss/ember-cli-postcss#switching-from-sass) for working with both sass and postcss in ember, but then I ran into [this issue](https://github.com/jeffjewiss/ember-cli-postcss/issues/143) a bunch and decided to try this other `filter` based approach.

HOWEVER, I just realized that the point of that method given in that guide is to _replace_ your sass pipeline with like a postcss-based one. So I tried that method along with _removing_ `ember-cli-sass`, and that seems to allow that method to work. So it seems to be a viable option, and might be the way to go.

Reply:
Oh, that's a great research! Thanks for the explanation :)

Reply:
:smile: